src/middlewares/officeMiddleware.js

```diff
@@ -0,0 +1,20 @@
+import office from '../db/officeDb'
+
+class officeValidator {
+    static createOffice(req,res,next){
+        try {
+            const {name, type} = req.body
+            if (!name) throw "name field required"
+            if (!type) throw 'type required'
```

Comment:
here also. Don't throw the error, instead, return the error message as a status.